### ISSUE TITLE
Option to select columns

### ISSUE THREAD

**Issue by ruanltbg** [emsk/bundle_outdated_formatter#3]
It would be good if this gem had an option to select which column you want to export it also would be great if you could order the columns.

**Comment by emsk** [comment]
Hello, @ruanltbg.

Thank you for the nice suggestion :smile:
I'd like to add `--column` option. (alias: `-c`, default: `gem newest installed requested groups`)

Usage:
```sh
$ bundle outdated | bof --column newest requested gem
┌────────┬───────────┬──────────┐
│ newest │ requested │ gem      │
├────────┼───────────┼──────────┤
│ 1.6.6  │ ~> 1.4    │ faker    │
│ 3.4.6  │ = 1.2.0   │ hashie   │
│ 2.3.1  │           │ headless │
└────────┴───────────┴──────────┘
```

How is this?

**Comment by ruanltbg** [comment]
@emsk it would be perfect. With only one command you can filter and sort the columns. 
Let me know if you need any help on this.

**Comment by emsk** [comment]
@ruanltbg Sorry for the late reply.
Released v0.6.0 includes this change.
Please try it 😄
